src/style.css

```
@@ -0,0 +1,120 @@
+* {
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  font-style: normal;
+  src: url("./fonts/Avenir-Book.ttf") format("truetype");
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 600;
+  font-style: normal;
+  src: url("./fonts/Avenir-Heavy.ttf") format("truetype");
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+.search {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+}
+
+.search__label__big {
+  width: 100%;
+  height: 70px;
+}
+
+.search__label__small {
+  width: 100%;
+  height: 42px;
+}
```

```suggestion
.search__label {
  width: 100%;
  height: 70px;
}

.search__label--small {
  height: 42px;
}
```
search__label будет у обоих label-ов, а --small только у маленького